metaserver/src/main/scala/scala/meta/languageserver/providers/DefinitionProvider.scala

```diff
@@ -15,13 +15,13 @@ object DefinitionProvider extends LazyLogging {
       uri: Uri,
       position: l.Position,
       tempSourcesDir: AbsolutePath
-  ): DefinitionResult = {
+  ): List[Location] = {
```

Comment:
How do you determine what data structures to return from the language server - raw payload, `XxxResult` or `XxxResponse`?

Reply:
Those as specified in https://microsoft.github.io/language-server-protocol/specification